docs/ref/models/fields.txt

```diff
@@ -502,6 +502,15 @@ Any combination of these options will result in an error.
     ``True`` will cause the field to have ``editable=False`` and ``blank=True``
     set.
 
+.. note::
+    When using timezones, storing a ``DateField`` in the database is
+    ambiguous because it's incapable of storing timezone aware information, unlike ``DateTimeField``.
```

Comment:
Oops, my comment was meant to go on the following comment. Serves me right for commenting from my phone.